packages/UI/docs/pdf_viewer.md

```diff
@@ -0,0 +1,67 @@
+# PDF Signer Web Component Documentation
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Usage](#usage)
+   <!-- - [Importing the PDF Signer](#importing-the-pdf-signer) -->
+   - [Initializing the PDF Signer](#initializing-the-pdf-signer)
+   - [Example of Usage](#example-of-usage)
+
+## 1. Introduction
+
+The PDF Signer Web Component is a straightforward PDF viewer that loads the PDF file into a single-page browser interface, allowing users to sign the data within the PDF and send it to an endpoint specified by the user.
+
+**Key features:**
+
+- Supports loading and viewing PDF files by providing the PDF URL.
+
+## 2. Usage
+
+<!-- ### Importing the PDF Signer
+
+Import the PDF Signer Web Component into your JavaScript file.
+
+```js
+import pdfSigner from "@threefold/UI";
+``` -->
+
+### Initializing the PDF Signer
```

Comment:
this the build instruction, not initialization

Reply:
changed